File: app/src/main/java/knf/kuma/recommended/RIHolder.java
package knf.kuma.recommended;

import android.view.View;
import android.widget.ImageView;
import android.widget.TextView;

import androidx.cardview.widget.CardView;
import androidx.recyclerview.widget.RecyclerView;
import butterknife.BindView;
import butterknife.ButterKnife;
import knf.kuma.R;

/**
 * Created by jordy on 26/03/2018.
 */

public class RIHolder extends RecyclerView.ViewHolder {
    @BindView(R.id.card)
    public CardView cardView;
    @BindView(R.id.img)
    public ImageView img;
    @BindView(R.id.title)
    public TextView title;
    @BindView(R.id.type)
    public TextView type;

    public RIHolder(View itemView) {
        super(itemView);
        ButterKnife.bind(this, itemView);
    }
}

File: app/src/main/java/knf/kuma/recommended/RIHolder.kt
package knf.kuma.recommended

import android.view.View
import android.widget.ImageView
import android.widget.TextView
import androidx.recyclerview.widget.RecyclerView
import com.google.android.material.card.MaterialCardView
import knf.kuma.R
import knf.kuma.commons.bind

/**
 * Created by jordy on 26/03/2018.
 */

class RIHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
    val cardView: MaterialCardView by itemView.bind(R.id.card)
    val img: ImageView by itemView.bind(R.id.img)
    val title: TextView by itemView.bind(R.id.title)
    val type: TextView by itemView.bind(R.id.type)
}
